test/unit/core/algorithm/pipeable_config_element_selector_test_template.hpp

```diff
@@ -0,0 +1,80 @@
+// -----------------------------------------------------------------------------------------------------
```

Comment:
The file seems unrelated?
Or did I miss something?

Reply:
ops!